tests/CMakeLists.txt

```diff
@@ -2,30 +2,8 @@ set(MIGRATION_FILES_LOCATION_V16 "${CMAKE_CURRENT_BINARY_DIR}/resources/v16/migr
 set(MIGRATION_FILES_LOCATION_V201 "${CMAKE_CURRENT_BINARY_DIR}/resources/v201/migration_files")
 set(DEVICE_MODEL_DB_LOCATION_V201 "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/everest/modules/OCPP201/device_model_storage.db")
 
-add_executable(database_tests database_tests.cpp)
-
-target_link_libraries(database_tests PRIVATE
-    ocpp
-    date::date
-    nlohmann_json::nlohmann_json
-    nlohmann_json_schema_validator
-    GTest::gtest_main
-    pthread
-    )
-
-
-add_test(database_tests database_tests)
-
 add_executable(libocpp_unit_tests)
 
-target_compile_definitions(database_tests
```

Comment:
The libocpp_unit_tests still pass without those compile definitions? Why were they needed anyway then?